spec/mailers/user_mailer_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe UserMailer, type: :mailer do
+  subject(:mail) { UserMailer.sign_in_email(user, sign_in_link) }
+  let(:user) { create(:user) }
+
+  let(:sign_in_link) do
+    Rails.application.routes.url_helpers.email_confirmation_url(
+      login_token: user.login_token, host: Rails.application.config.domain,
+    )
+  end
+
+  describe "#sign_in_email" do
+    it "renders the right headers and content" do
+      expect(mail.subject).to eq("Sign in into ECF")
+      expect(mail.to).to eq([user.email])
+      expect(mail.from).to eq(["from@example.com"])
+      expect(mail.body.encoded).to include(sign_in_link)
```

Comment:
Does that test pass now that you moved to use notify templates? It would be a little surprising to me - after all, how would our tests know the email body? Make an api call to notify, using a fake key, to get it? :/

Reply:
Good catch! I've updated it